### DISCUSSION TITLE
Getting as CSRF token authenticity error

### DISCUSSION THREAD

**Discussion by abhishek-triodec** [activeadmin/activeadmin#8459] [Help]
please more reference see it: https://github.com/oivoodoo/devise_masquerade/issues/144

**Comment by tagliala** [comment]
Hello, I'm incline to close this discussion because:
- It does not seem related
- It mentions [devise_masquerade](https://github.com/oivoodoo/devise_masquerade), which is not a requirement (also devise isn't) of Active Admin
- Does not contain the minimum information to provide info (Rails version, Ruby version, Active Admin version)
- I've tested v4 beta against Rails 7.2 and `test.local` domain and it works

**Comment by mayank-its** [comment]
Thanks for getting me back . I have solved this. could you help me to solve another error:
for more reference for this problem, please click on : https://discuss.rubyonrails.org/t/configuring-cookie-sharing-and-session-management-across-subdomains-in-rails-with-devise-masquerade/86503

**Reply by tagliala** [reply]
Sorry, this is off topic. Do you need a single rails application to work on multiple subdomains?

Maybe you are looking for https://guides.rubyonrails.org/configuring.html#actiondispatch-hostauthorization
